app/views/users/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<%# Regular Form %>
+
+<!--
+<form action="/users" method="post" accept-charset="UTF-8">
+  <label for="name">Username:</label>
+
+  <input type="text" id="name" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type = "hidden" name = "authenticity_token" value = "<%=form_authenticity_token%>">
+  <input type="submit" value="Submit">
+</form>
+-->
+
+
+<%# Form_tag %>
+
+<!--
+<%= form_tag("/users", method: "post") do %>
```

Comment:
Kindly use `users_path` from Rails helpers instead of `/users`

Reply:
Finished! Thanks!